sqlglot/generator.py

```diff
@@ -2322,6 +2324,27 @@ def tochar_sql(self, expression: exp.ToChar) -> str:
 
         return self.sql(exp.cast(expression.this, "text"))
 
+    def dictproperty_sql(self, expression: exp.DictProperty) -> str:
+        this = self.sql(expression, "this")
+        kind = self.sql(expression, "kind")
+        settings_sql = self.expressions(expression, key="settings", sep=" ")
+        args = f"({self.sep('')}{settings_sql}{self.seg(')', sep='')}" if settings_sql else "()"
+        return f"{this}({kind}{args})"
+
+    def dictrange_sql(self, expression: exp.DictRange) -> str:
+        this = self.sql(expression, "this")
+        max = expression.args.get("max")
```

Comment:
why get min and max? can you jsut do

min = self.sql(expression, "min")
max = self.sql(expression, "max")

Reply:
I'm not sure it's good to compare the string representation here.

Reply:
why not?

Reply:
You have created `def __eq__()` for a reason, if everybody would compare only strings, what's the point? :)

Reply:
But I can change to string here, as these are pretty simple, just literals.

Reply:
right, it's just the literal and should simplify the code